wasp-ai/src/server/operations.ts

```diff
@@ -53,6 +58,22 @@ export const startGeneratingNewApp: StartGeneratingNewApp<
   return appId;
 };
 
+export const registerZipDownload: RegisterZipDownload<{
+  appId: string;
+}> = async (args, context) => {
+  const appId = args.appId;
+  try {
+    await context.entities.Project.update({
+      where: { id: appId },
+      data: {
+        zipDownloadedAt: new Date(),
+      },
+    });
+  } catch (e) {
+    throw new HttpError(404, "App not found.");
+  }
```

Comment:
This is not great @sodic because you could be hiding quite a few errors this way!

Reply:
Ok I fixed it! Also fixed another occurrence of this in the same file, where you probably copied this from.

Reply:
Where copilot copied it from*. I've written 5 characters and pressed tab three times.